contracts/src/v0.8/test/HeartbeatRequester.t.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "forge-std/Test.sol";
```

Comment:
With these files in your own folder, please create a baseTest test that does some setup that will be useful to all of your like setting up an OWNER account and maybe a STRANGER, like you did with USER here. The OWNER will be the default deployer for all contracts, the STRANGER the default non-permissioned entity that shouldn't be allowed any special privileges. I guess you could use USER for that but I feel USER should be non-malicious as well. I'd expect STRANGER to check all your permissioned public/external calls for access control.

Reply:
This should also set a sensible block.timestamp that is known so something like this might work
```
vm.startPrank(OWNER);
deal(OWNER, 1e20);
vm.warp(BLOCK_TIME);
```

Reply:
This baseTest should be imported in every test and you should never import `"forge-std/Test.sol";` directly anymore.